src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const errorMessage = 'Bzzz... Error!';
```

Comment:
Static constants are usually named in uppercase and can be taken outside the function definition.